examples/call_a_tool_directly_with_auth.py

```diff
@@ -0,0 +1,51 @@
+"""
+This example demonstrates how to directly call a tool that requires authorization.
```

Comment:
Should this example be `star the ArcadeAI/arcade-ai repo` like the equivalent LLM one?